CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ If you do have such headers configured, then you must update the header configur
 - server: add logs for action handlers
 - server: add request/response sizes in event triggers (and scheduled trigger) logs (#5463)
 - server: change startup log kind `db_migrate` to `catalog_migrate` (#5531)
+- console: add pagination on rawsql results page (close #5002) (#5629)
```

Comment:
Can you take this to `Next release` section pls.

Reply:
oops, sorry changing now... :)